src/main/java/pl/psnc/dei/service/EuropeanaRestService.java

```diff
@@ -57,8 +57,7 @@ private void init() {
     public String postTranscription(Transcription transcription) {
         String annotationId = webClient.post()
```

Comment:
No need to store annotation id in local variable which is immediately returned.

Reply:
Yeah, i agree.
I made this variable, so that it will instantly tell what is returned. But as i made this comment above, there is no need for this anymore, so i will delete this variable, and return it immediately.